### PR TITLE
Filter

### PR DESCRIPTION
**Pull request by steveoni**
This pull request creates a filter process as suggested in this issue #3 .

Screen Shot 2021-03-01 at 19 52 14

### CODE REVIEW

File: src/Filter.js [hunk 1]
```diff
@@ -10,20 +10,143 @@ function Filter({ dataset, schema, filter, setFilter }) {
   // getTotalRows.filter({where: {ConnectedArea: {_eq: 'DK1'}}});
   const QUERY = gql`
     query Dataset {
-      ${getTotalRows}
+      ${getTotalRows}
```
Comment (anuveyatsu) [inline src/Filter.js:13]: it's very minor but we don't need extra whitespace here 😄

File: src/TableContainer.js [hunk 2]
```diff
@@ -5,21 +5,32 @@ const Query = require('graphql-query-builder');
...
   const QUERY = gql`
     query Dataset {
-      ${datasetQuery}
+      ${queryString}
     }
   `;
 
   const { loading, error, data } = useQuery(QUERY);
-
+
```
Comment (anuveyatsu) [inline src/TableContainer.js:33]: same here

File: src/TableContainer.js [hunk 3]
```diff
@@ -5,21 +5,32 @@ const Query = require('graphql-query-builder');
 
 
 function TableContainer({ dataset, schema, filter }) {
+  console.log(filter)
```
Comment (anuveyatsu) [inline src/TableContainer.js:8]: let's remove console logs in general

**Review by anuveyatsu — CHANGES_REQUESTED** [review]
@steveoni I've left some minor comments. Could you add tests for this feature? Let's have tests in general for all new stuff we're adding.

File: src/Filter.js (hunk 1 above)
Comment (steveoni) [reply src/Filter.js:13]: ok

**Pull request merged**